admin/services/data-access/check-form.data.service.js

```diff
@@ -48,10 +48,24 @@ const checkFormDataService = {
     return CheckForm.find(q).sort(sort).lean().exec()
   },
 
+  /**
+   * Create.
+   * @param data
+   * @returns {Promise<*>}
+   */
   create: async (data) => {
     const checkForm = new CheckForm(data)
     await checkForm.save()
     return checkForm.toObject()
+  },
+
+  /**
+   * Find check form by name.
+   * @param formName
+   */
+  findCheckFormByName: (formName) => {
+    let query = { 'isDeleted': false, 'name': formName }
+    return CheckForm.findOne(query).exec()
```

Comment:
This must return a plain javascript object please:
`CheckForm.findOne(query).lean().exec()` would work